src/sdft_implementations.jl

```diff
@@ -0,0 +1,54 @@
+export SDFT
+
+## Basic SDFT
+
+"""
+    SDFT(n)
```

Comment:
Can we maybe add here a minimal example with a iteration to see how it works?
Otherwise we can merge!